.github/workflows/neko.yml

```diff
@@ -0,0 +1,16 @@
+name: Qiskit Neko Integration Tests
+on:
```

Comment:
Is there something incorrect here? On this PR its showing two test sets being run, one for "push" and one for "pull_request". I would expect there to just be 1 per PR, and the other to run after merging to main.

Reply:
It's because I pushed this from a branch on the upstream (to get GHA to see it, it'd skip the job on a PR from a fork). So it's running on the commit to the `add-neko-tests` branch and also for this PR. For normal PRs opened from a fork this will only run on the PR and then again post-merge on the push to main or stable.